python/pyarrow/_fs.pyx

```diff
@@ -166,19 +186,39 @@ cdef class FileInfo(_Weakrefable):
         Returns
         -------
         type : FileType
+
+        Examples
+        --------
+        >>> local = getfixture('local_fs')
```

Comment:
> If we could just use the name `local` in all the examples, that would be much clearer.

I think that should actually be possible, if you insert "local" in the doctest_namespace. 
Of course, while that avoids the confusing `getfixture`, it still uses an object that is not defined in the docstring itself, making it harder to understand the docstring on its own (and to run the example yourself).

Trying to think for alternatives: specifically for the `FileInfo` class, it might also be OK to put all the examples in the main class docstrings, instead of in each attribute docstring? (we could still include a mention about that in each attribute docstring). That would also avoid the repeated creation of `local` + writing a single file.

Reply:
As all the main docstrings have examples that do not use any fixtures I presume that if someone is looking for example of that class's attribute, they would understand what local means as it is used in the example of the class itself. Maybe? =)